pkg/kubelet/rkt/rkt.go

```diff
@@ -425,6 +436,72 @@ func (r *Runtime) makePodManifest(pod *api.Pod, pullSecrets []api.Secret) (*appc
 	var globalPortMappings []kubecontainer.PortMapping
 	manifest := appcschema.BlankPodManifest()
 
+	listReq := &rktapi.ListPodsRequest{
+		Filter: &rktapi.PodFilter{
```

Comment:
can we make this filter a variable? or creating a function to return the filter ? Embedding it here hurts the readability.